Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/ImageClassifier/ImageClassifier.xaml.cs

```diff
@@ -0,0 +1,448 @@
+using WinMLSamplesGallery.Common;
+using WinMLSamplesGallery.Controls;
+using Windows.Foundation.Metadata;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation.Collections;
+using Windows.Foundation;
+
+namespace WinMLSamplesGallery.Samples
+{
+    public enum Classifier
+    {
+        NotSet = 0,
+        MobileNet,
+        ResNet,
+        SqueezeNet,
+        VGG,
+        AlexNet,
+        GoogleNet,
+        CaffeNet,
+        RCNN_ILSVRC13,
+        DenseNet121,
+        Inception_V1,
+        Inception_V2,
+        ShuffleNet_V1,
+        ShuffleNet_V2,
+        ZFNet512,
+        EfficientNetLite4,
+    }
+
+    public sealed class ClassifierViewModel
+    {
+        public string Title { get; set; }
+        public Classifier Tag { get; set; }
+    }
+
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class ImageClassifier : Page
+    {
+        const long BatchSize = 1;
+        const long TopK = 10;
+
+        private LearningModelSession _inferenceSession;
+        private LearningModelSession _postProcessingSession;
+        private LearningModelSession _preProcessingSession;
+        private LearningModelSession _tensorizationSession;
+
+        private Dictionary<Classifier, string> _modelDictionary;
+        private Dictionary<Classifier, Func<LearningModel>> _postProcessorDictionary;
+        private Dictionary<Classifier, Func<LearningModel>> _preProcessorDictionary;
+
+        private static Dictionary<long, string> _labels;
+        private static Dictionary<long, string> _imagenetLabels;
+        private static Dictionary<long, string> _ilsvrc2013Labels;
+
+
+        private BitmapDecoder CurrentImageDecoder { get; set; }
+
+        private Classifier CurrentModel { get; set; }
+
+        private Classifier SelectedModel
+        {
+            get
+            {
+                if (AllModelsGrid == null || AllModelsGrid.SelectedItem == null) {
+                    return Classifier.NotSet;
+                }
+                var viewModel = (ClassifierViewModel)AllModelsGrid.SelectedItem;
+                return viewModel.Tag;
+            }
+        }
+
+#pragma warning disable CA1416 // Validate platform compatibility
+        private LearningModelDeviceKind SelectedDeviceKind
+        {
+            get
+            {
+                return (DeviceComboBox.SelectedIndex == 0) ?
+                            LearningModelDeviceKind.Cpu :
+                            LearningModelDeviceKind.DirectXHighPerformance;
+            }
+        }
+#pragma warning restore CA1416 // Validate platform compatibility
+
+        public ImageClassifier()
+        {
+            this.InitializeComponent();
+
+            CurrentImageDecoder = null;
+            CurrentModel = Classifier.NotSet;
+            AllModelsGrid.SelectRange(new Microsoft.UI.Xaml.Data.ItemIndexRange(0, 1));

```

Comment:
isn't "Microsoft.UI.Xaml" already in the list of used namespaces?

Reply:
ItemIndexRange comes from the Microsoft.UI.Xaml.Data namespace which is not currently being included. I'll add a using statement.